### PR TITLE
Update ruby and rails

### PR DESCRIPTION
**Pull request by marcelo140**
The current ruby version is quite difficult to install (problems with docs and openssl) and gives me segmentation fault when I try to change the CSS. Since rails 4 has some problems with ruby 2.4 (Fixnum was deprecated), I ended up bumping the rails version too.

I tested in staging all our common operations (login, logout, create & edit activities, registrate, unregistrate, confirm/cancel confirmation) and, after some changes, I made it work. Still, the fact that we have no tests doesn't give me much confidence about this PR :S.

### CODE REVIEW

**Comment by fribmendes** [comment]
Before I review this, two questions:

1. Do you want to wait for specs?
2. Did you follow an upgrade guide? There are things that are now assumed as default in Rails 5, in terms of project directory.

**Comment by marcelo140** [comment]
1. I don't think the app would break, but I do prefer to wait for the specs since (hopefully) they're not that far away.

2. While I was upgrading I actually passed by the upgrade guide but forgot to stick to it. Thank you for reminding me.

File: app/models/application_record.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ApplicationRecord
```
Comment (fribmendes) [inline app/models/application_record.rb]: ⚠️

**Comment by marcelo140** [comment]
Done. Fixes https://github.com/cesium/atomic/issues/129 and https://github.com/cesium/atomic/issues/147. @fribmendes will you want to review this or do you prefer a let's just break it approach? I'm all for it :)

**Comment by fribmendes** [comment]
Let's just break it 😉

**Pull request merged**